### PR TITLE
Add missions to remind the player to check spaceports

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
Adds a new set of missions for players who ignore the spaceport. One is a spaceport mission that gets offered whenever the player checks the spaceport and has no other valid jobs available, and which updates a record of the last time that job was offered. The second is a mission that triggers when the game date is a set number of months after the recorded date. I decided to use the system implemented in this PR over checking to see if the player has checked the spaceport only once runs afoul of the current issue with spaceport visibility, where the player checks it once, then never does it again.

### CODE REVIEW

**Comment by Zitchas** [comment]
That's an amusing little encounter. I like it.

File: data/human/human missions.txt [hunk 1]
```diff
@@ -3764,3 +3764,57 @@ mission "Thrall Emigration"
...
+				`	"Nothing interesting happened the last time I checked a spaceport."`
+				`	"Why should I listen to you? I can handle myself, thank you very much."`
+					goto rebuff
+			`	The captain sighs. "Yeah, it can feel a little pointless running up against the same loud-mouthed people all the time. But just because a spaceport doesn't have anything to offer one day doesn't mean that it'll never have anything worthwhile. Wait a while, and there might be something new."`
+			choice
+				`	"Alright, I'll try to check more spaceports."`
+					goto thanks
+				`	"And why should I believe you over my own experiences?"`
+			label rebuff
+			`	He raises her hand and holds it open on his chest. "I know that it's only my word against yours, but I promise that you won't regret checking the spaceport every time you're in town."`
```
Comment (LepRyot) [inline data/human/human missions.txt:3810]:
```suggestion
			`	He raises his hand and holds it open on his chest. "I know that it's only my word against yours, but I promise that you won't regret checking the spaceport every time you're in town."`
```
Little thing here, good mission

**Review by LepRyot — CHANGES_REQUESTED** [review]

File: data/human/human missions.txt [hunk 2]
```diff
@@ -3764,3 +3764,57 @@ mission "Thrall Emigration"
...
+			choice
+				`	"Fine, I'll follow your advice."`
+					goto thanks
+				`	"I don't need your help."`
+			`	A frown creeps onto the captain's face as he earnestly says, "Well, I'm sorry that my advice wasn't helpful. Good luck, Captain." He walks off, leaving you alone at the job board.`
+				decline
+			label thanks
+			`	He pleasantly smiles and nods. "Well, I wish you the best in your travels." He walks away, leaving you free to explore the rest of <planet>'s facilities.`
```
Comment (Amazinite) [inline data/human/human missions.txt:3818]: Both replacements yield the same results in this case, but it's more proper to use `<origin>` when referring to the offering planet (plus you already used `<origin>` in the first line).
```suggestion
			`	He pleasantly smiles and nods. "Well, I wish you the best in your travels." He walks away, leaving you free to explore the rest of <origin>'s facilities.`
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Comment by MasterOfGrey** [comment]
Is it possible to have this turn into a long reminder afterwards?
Like, if you've already had this interaction, keep resetting the timer occasionally and if you don't visit a spaceport for say, twice as long as the gap for this encounter, you get an `on landing` "As you ignore the spaceport for the umpteenth time a little voice in your head says, 'you won't regret checking the spaceport every time you're in town...'"
And then just have this repeat infinitely every time you go that long without checking?

**Comment by Amazinite** [comment]
Possible? Yeah. Should we do it? I'd rather not.

**Pull request merged**